File: c/c-struct/test2.c
#include <stdlib.h>
#include <stdio.h>

typedef struct Personne Personne;

struct Personne {
	char *name;
	Personne *p;
};


int main(){
	Personne p1;
	Personne p2;

	p1.name = "hello";
	p2.name = "world";

	p1.p = &p2;

	printf("%s\n", p1.p->name);

	return 0;
}

File: python/python-qrcode/test2.py
#!/usr/bin/python

import PyQRNative
 
def makeQR(data_string,path,level=1):
    quality={1: PyQRNative.QRErrorCorrectLevel.L,
             2: PyQRNative.QRErrorCorrectLevel.M,
             3: PyQRNative.QRErrorCorrectLevel.Q,
             4: PyQRNative.QRErrorCorrectLevel.H}
    size=3
    while 1:
        try:
            q = PyQRNative.QRCode(size,quality[level])
            q.addData(data_string)
            q.make()
            im=q.makeImage()
            im.save(path,format="png")
            break
        except TypeError:
            size+=1

if __name__ == "__main__":
	makeQR("Hello world !", ".");
